nucypher/blockchain/eth/actors.py

```diff
@@ -102,8 +104,8 @@ def __repr__(self):
     @property
     def eth_balance(self) -> int:
```

Comment:
This returns a `Decimal` not an `int`. The other option is returning the result in `wei` (I strongly prefer this).

Reply:
updated the annotation for now. :heavy_check_mark: